res/values/styles.xml

```diff
@@ -23,6 +23,7 @@
 
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit